clang/lib/Headers/aiev2_aie_api_compat.h

```diff
@@ -1398,6 +1443,10 @@ inline __attribute__((always_inline)) v16cbfloat16 sel(v16cbfloat16 , v16cbfloat
 inline __attribute__((always_inline)) unsigned int le(v16float , v16float );
 inline __attribute__((always_inline)) unsigned int eq(v16float , v16float );
 inline __attribute__((always_inline)) unsigned int ne(v16float , v16float );
+inline __attribute__((always_inline)) void * add_2d_int(void * , int , int , addr_t , addr_t , addr_t & );
+inline __attribute__((always_inline)) void * add_2d_byte_int(void * , int , int , addr_t , addr_t , addr_t & );
+inline __attribute__((always_inline)) void * add_3d_int(void * , int , int , int , addr_t , addr_t , addr_t & , addr_t , addr_t , addr_t & );
+inline __attribute__((always_inline)) void * add_3d_byte_int(void * , int , int , int , addr_t , addr_t , addr_t & , addr_t , addr_t , addr_t & );
```

Comment:
Curious: What do we define `addr_t` to, and does our calling convention support it?

Reply:
`addr_t` is `typedef`'d to `int32_t`.